lib/jsdom/living/named-properties-window.js

```diff
@@ -11,7 +11,7 @@ function isNamedPropertyElement(element) {
 
   // use hasOwnProperty to make sure contentWindow comes from the prototype,
   // and is not set directly on the node by a script.
-  if ("contentWindow" in element && !hasOwnProp.call(element, "contentWindow")) {
+  if ("contentWindow" in element && !hasOwn(element, "contentWindow")) {
```

Comment:
Let's not add a separate util for this. Instead just undo these changes; they're not related to this PR and we're happy with the current style.